src/clr-addons/daterangepicker/directives/daterangepicker-container/daterangepicker-container.component.html

```diff
@@ -0,0 +1,79 @@
+<ng-content select="label"></ng-content>
+<div class="clr-col-12 clr-col-md-10 clr-control-container" [class.clr-error]="this.isInvalid">
+  <div class="clr-input-wrapper" clrPopoverAnchor>
+    <div class="clr-input-group" [class.clr-focus]="this.focus">
+      <ng-content select="[clrDaterangepicker]"></ng-content>
+      <button
+        type="button"
+        class="clr-input-group-icon-action"
+        [disabled]="this.disabled"
+        [title]="this.friendlyDaterange"
+        [attr.aria-label]="this.friendlyDaterange"
+        clrPopoverOpenCloseButton
+      >
+        <cds-icon status="info" shape="calendar"></cds-icon>
```

Comment:
This calendar has no cursor:pointer

Reply:
Funny enough, neither does Clarity Datepicker; `cursor: default`.

But I'll add it anyways.

Reply:
I would actually prefer to go with consistency here and keep it like this until Clarity changes their cursor for datepicker. 
I opened an issue at their repo: https://github.com/vmware-clarity/ng-clarity/issues/559